pyproject.toml

```diff
@@ -20,8 +20,8 @@ classifiers = [
 requires-python = ">=3.9, <3.12"
 dependencies = [
     "numpy",
-    "qiskit >= 0.45.1, < 0.46",
-    "qiskit-aer >= 0.13.1, < 0.14",
+    "qiskit >= 0.45, < 1.2",
```

Comment:
Should we allow also Qiskit 1.2? At least it's already included in the test environments.

Reply:
It does not work because of the deprecation of BackendProperties that is used in the Deneb transpiler. Ive added qiskit v1.2 support to the transpiler improvements ticket so these changes can be made sequentially.

Reply:
Then I think `qiskit-v1.2` should be removed from `tox.ini` environments for now.